CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- #4704 - Adds ability to show a different alpha and disables checkbox
```

Comment:
It's not clear where this change affects the app. Perhaps something like "Delete browsing data items will show a different alpha when there is nothing to delete" would be more clear.

Reply:
Agreed, there's a bit of ambiguity here, let me refactor this and push again.

Reply:
Pushed change to include this.